spot-client/src/common/app-state/remote-control-service/actions.js

```diff
@@ -59,6 +71,81 @@ function createActionWithRequestStates( // eslint-disable-line max-params
         });
 }
 
+/**
+ * Establishes a connection to an existing Spot-MUC using the provided join code.
+ *
+ * @returns {Object}
+ */
+export function createSpotTVRemoteControlConnection() {
```

Comment:
I'm thinking maybe this should not be placed in the 'common' package since it's Spot TV specific

Reply:
I'll have to think about how to move this action and "createActionWithRequestStates".